src/Uno.Extensions.Reactive.UI/Uno.Extensions.Reactive.UI.WinUI.csproj

```diff
@@ -10,6 +10,8 @@
 		<EnableDefaultPageItems>false</EnableDefaultPageItems>
 		<DefineConstants>$(DefineConstants);WINUI</DefineConstants>
 		<PackageId>Uno.Extensions.Reactive.WinUI</PackageId>
+		
+		<UnoXamlResourcesTrimming Condition="'$(MSBuildRuntimeType)'=='Core' and '$(Configuration)'=='Release'">true</UnoXamlResourcesTrimming>
```

Comment:
@ebariche @jeromelaban should we be making this adjustment to Navigation.UI and Navigation.Toolkit.UI libraries?

Reply:
It's only needed if you have XAML files in libraries. I did not find any, but I may have missed it.